03.bowling/bowling.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
```

Comment:
```suggestion
  s == 'X' ? shots << 10 << 0 : shots << s.to_i
```

ちょっとトリッキーですがこんなふうにも書けます

Reply:
条件演算子を使用して、条件式を1行にまとめました。